packages/api/src/controllers/dashboard.ts

```diff
@@ -1,24 +1,58 @@
-import { differenceBy, uniq } from 'lodash';
+import { differenceBy, groupBy, uniq } from 'lodash';
 import { z } from 'zod';
 
-import { DashboardWithoutIdSchema, Tile } from '@/common/commonTypes';
+import { DashboardWithoutIdSchema } from '@/common/commonTypes';
 import type { ObjectId } from '@/models';
 import Alert from '@/models/alert';
 import Dashboard from '@/models/dashboard';
-import { tagsSchema } from '@/utils/zod';
 
 export async function getDashboards(teamId: ObjectId) {
-  const dashboards = await Dashboard.find({
+  const _dashboards = await Dashboard.find({
     team: teamId,
   });
+
+  const alertsByTileId = groupBy(
+    await Alert.find({
+      dashboard: { $in: _dashboards.map(d => d._id) },
+      source: 'tile',
+    }),
```

Comment:
double check we want to select all fields from the alert?

Reply:
yep, I think all data is fine, we need full `channel` to populate the form